src/bam/mod.rs

```diff
@@ -438,6 +452,12 @@ quick_error! {
         NoMoreRecord {
             description("no more record")
         }
+        FetchError {
+            description("bam fetch failed")
+        }
+        SeekError {
+            description("bam seek failed")
+        }
```

Comment:
That is of course correct and you define these errors below. My point is that they don't need to be additionally part of readerror. Or are you throwing them anywhere and I just don't see it on my mobile phone?